azure-functions/acbs-function/.prettierrc.json

```diff
@@ -0,0 +1,18 @@
+{
+  "arrowParens": "always",
+  "bracketSameLine": false,
+  "bracketSpacing": true,
+  "embeddedLanguageFormatting": "auto",
+  "htmlWhitespaceSensitivity": "css",
+  "insertPragma": false,
+  "jsxSingleQuote": false,
+  "printWidth": 160,
```

Comment:
A `printWidth` of 160 feels quite wide to me, and I often see functions and types that prettier formats to a wider width than I'd like. What your thoughts on adjusting this to something like 120?

Reply:
Good suggestion, have adjusted to `120`.